spec/acceptance/hieradata/common.yaml

```diff
@@ -5,4 +5,6 @@ foreman::server_ssl_cert: /etc/foreman-certs/certificate.pem
 foreman::server_ssl_chain: /etc/foreman-certs/certificate.pem
 foreman::server_ssl_crl: ""
 foreman::server_ssl_key: /etc/foreman-certs/key.pem
-foreman::user_groups: []
+foreman::client_ssl_ca: /etc/foreman-certs/certificate.pem
+foreman::client_ssl_cert: /etc/foreman-certs/certificate.pem
+foreman::client_ssl_key: /etc/foreman-certs/key.pem
```

Comment:
For this I thought about making these defaults undef and then `pick($server, $client)` to determine the real value. That way, you don't need to specify them twice if they match.

Reply:
I think I could tackle that over here -- https://github.com/theforeman/puppet-foreman/pull/951